File: AquariumApi.Models/Aquarium/DeviceInformation.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace AquariumApi.Models
{
    public class DeviceInformation
    {
        public ScheduleState ScheduleStatus;
        public string Version {get; set;}
        public object Config {get; set; }
        public AquariumUser Account { get; set; }
        public DeviceConfiguration ConfiguredDevice { get; set; }
    }
}

File: AquariumApi.DeviceApi/ClientApp/src/app/modules/SharedDeviceModule/models/DeviceInformation.ts
import { AquariumAccount } from './AquariumAccount'
import { DeviceConfiguration } from './DeviceConfiguration'
import { DeviceScheduleState } from './DeviceScheduleState'

export class DeviceInformation {
  version: string
  config: object
  account: AquariumAccount
  configuredDevice: DeviceConfiguration
  scheduleStatus: DeviceScheduleState
}

